src/test/bundles/12-exported-pkg-vars/uds-bundle.yaml

```diff
@@ -0,0 +1,28 @@
+kind: UDSBundle
+metadata:
+  name: export-vars
+  description: testing a bundle using exported vars from zarf package
+  version: 0.0.1
+
+packages:
+  - name: output-var
+    repository: localhost:888/output-var
+    ref: 0.0.1
+    exports:
+      - name: COLOR
+
+  - name: helm-overrides
+    path: "../../packages/helm"
+    ref: 0.0.1
+    imports:
+      - name: COLOR
+        package: output-var
+
+    overrides:
+      podinfo-component:
+        unicorn-podinfo:
+          values:
+            - path: "podinfo.replicaCount"
+              value: 1
+            - path: "podinfo.ui.color"
+              value: ${COLOR}
```

Comment:
Is there a reason this wouldn't be a `variable` with name `color` instead of `value`? This definitely works but feels a bit weird to me since I would typically think of `values` being "hardcoded" in a bundle and `variables` being dynamic.

Reply:
Good catch @mjnagel ! The vars are set at deploy time so the template probably doesn't make sense under `values`, only under `variables`

Reply:
I can switch it if that makes more sense, I just went with `values` because under `variables` it would get set under `default` since there is no value field for variables.

Reply:
Good point @decleaver , looks like the issue is to have this under `values` 🤔

Reply:
After our synchronous chat I'm on board with `values`. As @UncleGedd explained it `variables` is more for things that the bundle deployer can configure. This is more in line with `values` that aren't configurable by the bundle deployer.

Reply:
After discussing, going to stick with setting in `values` vs `variables`